coderd/coderd.go

```diff
@@ -1413,6 +1413,12 @@ func New(options *Options) *API {
 					r.Delete("/", api.deleteWorkspaceAgentPortShare)
 				})
 				r.Get("/timings", api.workspaceTimings)
+				r.Route("/acl", func(r chi.Router) {
+					r.Use(
+						httpmw.RequireExperimentWithDevBypass(api.Experiments, codersdk.ExperimentWorkspaceSharing))
+
+					r.Patch("/", api.patchWorkspaceACL)
```

Comment:
Can we make this just an experiment? I don't think it should pop onto a deployment unexpectedly yet.